### PR TITLE
Properties hash

### PR DESCRIPTION
**Pull request by yaichenbaum**

### CODE REVIEW

**Comment by duke7553** [comment]
image

File: Files/Interacts/Interaction.cs [hunk 1]
```diff
@@ -1182,5 +1187,19 @@ public void PushJumpChar(char letter)
         {
             App.CurrentInstance.ViewModel.JumpString += letter.ToString().ToLower();
         }
+
+        public async Task<string> GetHashForFile(ListedItem fileItem, string nameOfAlg)
```
Comment (yaichenbaum) [inline Files/Interacts/Interaction.cs:1191]: Can you rename this to `GetMD5HashFromFile`?

Comment (duke7553) [reply Files/Interacts/Interaction.cs:1191]: Actually, if we pass in an alternative second parameter, we can get different types of hashes.

Comment (duke7553) [reply Files/Interacts/Interaction.cs:1191]: That's why I left it as a generic name just in case.

Comment (yaichenbaum) [reply Files/Interacts/Interaction.cs:1191]: OK, that is very good.

**Pull request merged**